contrib/podman-tui/template.py

```diff
@@ -0,0 +1,20 @@
+pkgname = "podman-tui"
+pkgver = "1.2.0"
+pkgrel = 0
+build_style = "go"
+hostmakedepends = ["go"]
+makedepends = ["linux-headers", "libbtrfs-devel"]
+checkdepends = ["pkgconf", "gpgme-devel"]
+go_build_tags = [
+    "exclude_graphdriver_devicemapper",
+    "exclude_graphdriver_btrfs",
+    "btrfs_noversion",
```

Comment:
these are definitely not meant to be set

Reply:
hm well they're set by default, though in alpine they're not..

Reply:
let's assume it works

Reply:
i got them from the project's makefile.